powerauth-restful-server-javaee/src/main/java/io/getlime/security/powerauth/app/rest/api/javaee/controller/v2/EncryptedDataExchangeController.java

```diff
@@ -92,7 +93,7 @@
         try {
             requestDataBytes = encryptor.decrypt(request);
         } catch (GenericCryptoException | CryptoProviderException | InvalidKeyException ex) {
-            logger.warn(ex.getMessage(), ex);
+            logger.warn("Encryption failed, error: {}", ex.getMessage());
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.